services/venice-server/src/main/java/com/linkedin/venice/stats/ServerQuotaUsageStats.java

```diff
@@ -28,8 +23,6 @@ public ServerQuotaUsageStats(MetricsRepository metricsRepository, String name) {
     rejectedQPS = registerSensor("quota_rejected_request", new Rate());
     rejectedKPS = registerSensor("quota_rejected_key_count", new Rate());
     allowedUnintentionallyKPS = registerSensor("quota_unintentionally_allowed_key_count", new Count());
-    usageRatioSensor =
-        registerSensor(new AsyncGauge((ignored, ignored2) -> getReadQuotaUsageRatio(), "quota_requested_usage_ratio"));
```

Comment:
I removed this metric for two reasons: 1) it seems it's not accurate since usage is usually per sec and the metric reporting happens at minute level 2) We can infer the same info using quota request rate & quota rejected rate

Reply:
Thanks Xun for the clarification. As discussed offline I added back quota usage ratio metric that is calculated using the formula: `quota_request_key_count / nodeQpsResponsibility`